docs/packages/only_wiki/ARM_NEON_2_x86_SSE.rst

```diff
@@ -0,0 +1,24 @@
+.. spelling::
+
+    ARM_NEON_2_x86_SSE
+
+.. _pkg.ARM_NEON_2_x86_SSE:
+
+ARM_NEON_2_x86_SSE
+==================
+
+File only available in wiki.
+Please merge with hunter-package entry
```

Comment:
What does it mean?

Reply:
for this particular package it means, that my matching algorithm is wrong. For other packages it means, that for `pkg.<pkg_name>.md` there was no `cmake/projects/<pkg_name>` folder.

The markdown filename is assumed to be lower case, thats why `ARM_NEON_2_X86_SSE` is not found. For others the markdown filename and the hunter-folder name don't match

Reply:
> that for pkg.<pkg_name>.md there was no cmake/projects/<pkg_name> folder

Can you create the list with such packages, I will take a look.

Reply:
the full list of packages with no documentation vs wiki entries with no matching hunter-projects

```
only_hunter/AllTheFlopsThreads.rst
only_hunter/Clang.rst
only_hunter/ClangToolsExtra.rst
only_hunter/CLAPACK.rst
only_hunter/clBLAS.rst
only_hunter/CppNetlib.rst
only_hunter/crashup.rst
only_hunter/czmq.rst
only_hunter/damageproto.rst
only_hunter/dbus.rst
only_hunter/dri2proto.rst
only_hunter/dri3proto.rst
only_hunter/drishti_assets.rst
only_hunter/drishti_faces.rst
only_hunter/drm.rst
only_hunter/eigen3-nnls.rst
only_hunter/enet.rst
only_hunter/fixesproto.rst
only_hunter/geos.rst
only_hunter/getopt.rst
only_hunter/glew.rst
only_hunter/glproto.rst
only_hunter/ice.rst
only_hunter/Igloo.rst
only_hunter/inputproto.rst
only_hunter/intltool.rst
only_hunter/ios_sim.rst
only_hunter/ippicv.rst
only_hunter/kbproto.rst
only_hunter/LAPACK.rst
only_hunter/Leptonica.rst
only_hunter/Libcxx.rst
only_hunter/Libcxxabi.rst
only_hunter/libdaemon.rst
only_hunter/libjson-rpc-cpp.rst
only_hunter/libogg.rst
only_hunter/librtmp.rst
only_hunter/libxml2.rst
only_hunter/LLVMCompilerRT.rst
only_hunter/mini_chromium.rst
only_hunter/mpark_variant.rst
only_hunter/mtplz.rst
only_hunter/MySQL-client.rst
only_hunter/nanoflann.rst
only_hunter/NASM.rst
only_hunter/odb.rst
only_hunter/odb-boost.rst
only_hunter/odb-compiler.rst
only_hunter/odb-mysql.rst
only_hunter/odb-sqlite.rst
only_hunter/onmt.rst
only_hunter/OpenCV-Extra.rst
only_hunter/OpenNMTTokenizer.rst
only_hunter/pciaccess.rst
only_hunter/presentproto.rst
only_hunter/pthread-stubs.rst
only_hunter/pugixml.rst
only_hunter/QtCMakeExtra.rst
only_hunter/randrproto.rst
only_hunter/renderproto.rst
only_hunter/SDL_mixer.rst
only_hunter/sm.rst
only_hunter/Snappy.rst
only_hunter/Sober.rst
only_hunter/Sources-for-Android-SDK.rst
only_hunter/SQLite3.rst
only_hunter/SuiteSparse.rst
only_hunter/Tesseract.rst
only_hunter/tomcrypt.rst
only_hunter/tommath.rst
only_hunter/Washer.rst
only_hunter/WinSparkle.rst
only_hunter/x11.rst
only_hunter/x264.rst
only_hunter/xau.rst
only_hunter/xcb.rst
only_hunter/xcb-proto.rst
only_hunter/xcursor.rst
only_hunter/xdamage.rst
only_hunter/xext.rst
only_hunter/xextproto.rst
only_hunter/xf86vidmodeproto.rst
only_hunter/xfixes.rst
only_hunter/xi.rst
only_hunter/xinerama.rst
only_hunter/xineramaproto.rst
only_hunter/xorg-macros.rst
only_hunter/xproto.rst
only_hunter/xrandr.rst
only_hunter/xrender.rst
only_hunter/xshmfence.rst
only_hunter/xtrans.rst
only_hunter/xxf86vm.rst
only_hunter/zookeeper.rst
```

vs

```
only_wiki/accelerate.rst
only_wiki/appkit.rst
only_wiki/assetslibrary.rst
only_wiki/audiotoolbox.rst
only_wiki/avfoundation.rst
only_wiki/coredata.rst
only_wiki/corefoundation.rst
only_wiki/coregraphics.rst
only_wiki/corelocation.rst
only_wiki/coremedia.rst
only_wiki/corevideo.rst
only_wiki/egl.rst
only_wiki/foundation.rst
only_wiki/glapi.rst
only_wiki/gles2.rst
only_wiki/gles3.rst
only_wiki/glkit.rst
only_wiki/imageio.rst
only_wiki/iokit.rst
only_wiki/metal.rst
only_wiki/mobilecoreservices.rst
only_wiki/mysql.client.rst
only_wiki/opengles.rst
only_wiki/osmesa.rst
only_wiki/quartzcore.rst
only_wiki/uikit.rst
```